front/lib/actions/mcp_internal_actions/servers/run_agent.ts

```diff
@@ -162,7 +162,11 @@ export default async function createServer(
 
   server.tool(
     `run_${childAgentBlob.name}`,
-    `Run agent ${childAgentBlob.name} (${childAgentBlob.description})`,
+    `Run agent ${childAgentBlob.name} (${childAgentBlob.description}). ` +
```

Comment:
Big question on wether we want this here or with the same kind of logic we have on `constructPromptMultiActions` with `citationMetaPrompt`.

Reply:
Deal, adding a commit!

Reply:
Done!